### PR TITLE
Add qgis-plugin-manager package

### PR DESCRIPTION
**Pull request by yuvipanda**
Awaiting https://github.com/3liz/qgis-plugin-manager/pull/61, which packages the LICENSE file appropriately. 
In the meantime, added the LICENSE here.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by yuvipanda** [comment]
Is the linter confused? I definitely did not delete that file

**Comment by yuvipanda** [comment]
I just had it in the wrong place

File: recipes/qgis-plugin-manager/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - pip check
+    - qgis-plugin-manager --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/3liz/qgis-plugin-manager
+  summary: Tool for downloading/managing QGIS plugins from CLI.
+  dev_url: https://github.com/3liz/qgis-plugin-manager
+  license: 'GPL-3.0-only'
```
Comment (ocefpaf) [inline recipes/qgis-plugin-manager/meta.yaml:39]: Hi Yuvi, I see this file is in a PR upstream in https://github.com/3liz/qgis-plugin-manager/pull/61, right?
We need that to be merged first so we know upstream authors are OK with it before merging this PR.

Comment (yuvipanda) [reply recipes/qgis-plugin-manager/meta.yaml:39]: @ocefpaf thanks! I'll re-mark this as draft until that gets merged!

Comment (Gustry) [reply recipes/qgis-plugin-manager/meta.yaml:39]: I just merged the PR upstream. Thanks for this work.

**Comment by yuvipanda** [comment]
@conda-forge/help-python ready for review!

**Pull request merged**